### DISCUSSION TITLE
Can .net maui have better support for AppWidget in mobile segments?

### DISCUSSION THREAD

**Discussion by JeffCaptain** [dotnet/maui#16459] [Q&A]
I didn't find any support information about AppWidget in the official documentation. 
I asked some people and they told me that all the information came from xamrain. 
I tried to port the code to .net Maui, and it did create an AppWidget, but it didn't work properly and there were a lot of problems. 
And it looks very complex, can the Microsoft team better support AppWidget in. Net maui? 
Both IOS and Android 13 have good support and optimization for AppWidget. 
I hope .net Maui can also easily make AppWidget.

Take the AppWidget part in xamarin. There is a directory structure that can create an AppWidget, but it cannot be used
![image](https://user-images.githubusercontent.com/52701673/186217780-6c1bbef8-1230-47fb-a94a-1bc0da9d04ba.png)

**Comment by ghost** [comment]
We've moved this issue to the Backlog milestone. This means that it is not going to be worked on for the coming release. We will reassess the backlog following the current release and consider this item at that time. To learn more about our issue management process and to have better expectation regarding different types of issues you can read our [Triage Process](https://github.com/dotnet/maui/blob/main/docs/TriageProcess.md).
